logstash-core/lib/logstash/runner.rb

```diff
@@ -501,4 +512,15 @@ def setting(key)
     @settings.get_value(key)
   end
 
+  def verify_deprecated_setting(old_setting, new_setting)
+    if @settings.set?(old_setting)
+      logger.warn("Deprecated setting `#{old_setting}` please use `#{new_setting}`")
+      if @settings.set?(new_setting)
+        signal_usage_error("Both `#{new_setting}` and the deprecated `#{old_setting}` settings are set, please remove `#{old_setting}`")
+      else
```

Comment:
this `else` is the false-case of `if @settings.set?(new_setting)`; I think this clause is meant to be the false-case of the `if @settings.set?(old_setting)` clause.